available-connectors.json

```diff
@@ -191,6 +201,28 @@
     "supportedByStrongLoop": true
   },
   {
+    "name": "kv-redis",
+    "description": "Redis key-value connector",
+    "baseModel": "KeyValueModel",
+    "features": {
+      "discovery": false,
+      "migration": false
+    },
+    "settings": {
+      "host": {
+        "type": "string"
+      },
+      "port": {
+        "type": "number"
+      }
```

Comment:
Also added the `url` property because it seems like we'll need it for strongloop/loopback-connector-kv-redis#15.